scripts/dump-target-info.pl

```diff
@@ -5,6 +5,7 @@
 use Cwd;
 
 my (%targets, %architectures, %kernels);
+my @devices = ();
```

Comment:
@aparcar can you give me an example output? I'm not sure what we should use as ID.

Example from an output like this

```
@@

Target-Profile: DEVICE_tplink_ad7200
Target-Profile-Name: TP-Link Talon AD7200 v1/v2 (TP-Link AD7200 v1/v2)
Target-Profile-Packages: ath10k-firmware-qca99x0-ct kmod-ramoops kmod-wil6210
Target-Profile-hasImageMetadata: 1
Target-Profile-SupportedDevices: tplink,ad7200


Target-Profile-Description:
Build firmware images for TP-Link AD7200 v1/v2
Alternative device titles:
- TP-Link Talon AD7200 v1/v2





@@

Target-Profile: DEVICE_tplink_ad7200
Target-Profile-Name: TP-Link AD7200 v1/v2
Target-Profile-Packages: ath10k-firmware-qca99x0-ct kmod-ramoops kmod-wil6210
Target-Profile-hasImageMetadata: 1
Target-Profile-SupportedDevices: tplink,ad7200


Target-Profile-Description:
Build firmware images for TP-Link AD7200 v1/v2
Alternative device titles:
- TP-Link Talon AD7200 v1/v2





@@
```

the script produce

```
...
Qualcomm AP148 legacy
Qualcomm AP148 standard
Qualcomm AP161
Qualcomm DB149
TP-Link AD7200 v1/v2
TP-Link Archer C2600 v1
TP-Link Archer VR2600v v1
TP-Link Talon AD7200 v1/v2 (TP-Link AD7200 v1/v2)
Ubiquiti UniFi AC HD
ZyXEL NBG6817
```

Reply:
Alternative device titles are harder to handle since they appear to be multi line. Maybe @jow- knows how to deal with it.

```
Alternative device titles:
- Mojo Networks C-75
- WatchGuard AP320
```

Also you may want to use different variables names, i.e. device_name -> device_title, device_profile -> device

Reply:
@aparcar can you give me example where multiline alternative happen? So I can waste the day finding a good solution? ahahaha

Reply:
Sorry missing bits for actually printing stuff:

```diff
@@ -134,8 +143,8 @@ elsif (@ARGV == 1 && $ARGV[0] eq 'kernels') {
 }
 elsif (@ARGV == 2 && $ARGV[0] eq 'devices') {
        get_devices($ARGV[1]);
-       foreach my $device_name (sort values @devices) {
-               printf "%s\n", $device_name;
+       foreach my $device (sort keys %devices) {
+               printf "%s %s\n", $device, $devices{$device};
        }
 }
 else {
```

Reply:
> @aparcar can you give me example where multiline alternative happen? So I can waste the day finding a good solution? ahahaha

```
Target-Profile: DEVICE_airtight_c-75
Target-Profile-Name: Mojo Networks C-75 (AirTight Networks C-75)
Target-Profile-Packages: ath10k-firmware-qca988x-ct kmod-ath10k-ct kmod-usb2
Target-Profile-hasImageMetadata: 1
Target-Profile-SupportedDevices: airtight,c-75


Target-Profile-Description:
Build firmware images for AirTight Networks C-75
Alternative device titles:
- Mojo Networks C-75
- WatchGuard AP320
```